### PR TITLE
Improves the petition page layout (Issue 202)

### PR DESCRIPTION
**Pull request by viniciusvgv**
This pull request solves the issue #202 .It fixes some front-end mistakes and makes the petition page look more similar to the layout designed.

## How was it before?
Some things on the petition page had wrong dimensions and were misaligned

## What has changed?
I made some changes to make the implementation more similar to the prototype and some other improvements.

## What should I pay attention when reviewing this PR?
You should be sure the layout works fine on tablets, phones and desktops.

## Is this PR dangerous?
No

### Large
plip fixed

## Small
image

## Extra-small
image

### CODE REVIEW

File: app/assets/javascripts/mu.petition-signers.js [hunk 1]
```diff
@@ -34,8 +44,8 @@
       var addRow =  function($element, userInfo) {
         var $row = $("<li></li>");
 
-        var $img = $("<div class='pull-left'><img /></div>");
-        $img.find("img").attr("src", userInfo.profile_picture);
+        var $img = $("<div class='pull-left avatar'></div>");
+        $img.attr("style", "background-image: url('" + userInfo.profile_picture + "')");
```
Comment (guimello) [inline app/assets/javascripts/mu.petition-signers.js]: You can use the `#css` func.

```
$img.css("background-image", "url('xpto.png')");
```

File: app/assets/javascripts/mu.petition-signers.js [hunk 2]
```diff
@@ -51,21 +61,21 @@
...
-     
+      $element.append("<div class='container-fluid'><h3 class='title'>Assinantes recentes</h3><ul class='list-unstyled'></ul></div>");
+
       var addRow = function($element, userInfo) {
         var $row = $("<li></li>");
 
-        var $img = $("<div class='pull-left'><img /></div>");
-        $img.find("img").attr("src", userInfo.profile_picture);
+        var $img = $("<div class='pull-left avatar'></div>");
+        $img.attr("style", "background-image: url('" + userInfo.profile_picture + "')");
```
Comment (guimello) [inline app/assets/javascripts/mu.petition-signers.js]: here too

File: app/assets/stylesheets/global.sass [hunk 3]
```diff
@@ -3162,3 +3163,19 @@ nav.navbar.navbar-default ul.dropdown-menu.cycle
...
+  display: inline-block
+  line-height: 1
+  text-transform: none
+  letter-spacing: normal
+  word-wrap: normal
+  white-space: nowrap
+  direction: ltr
+  -webkit-font-smoothing: antialiased
+  text-rendering: optimizeLegibility
+  font-feature-settings: "liga"
```
Comment (guimello) [inline app/assets/stylesheets/global.sass]: Oh man, so advanced :)

File: app/assets/stylesheets/petition/index.css.sass [hunk 4]
```diff
@@ -0,0 +1,325 @@
+@import "bootstrap/variables"
```
Comment (guimello) [inline app/assets/stylesheets/petition/index.css.sass:1]: We're going to have some conflicts here :p

File: app/views/cycles/plugin_relations/petition.html.slim [hunk 5]
```diff
@@ -26,105 +26,105 @@ section#petition-index
...
-        p= @petition.presentation
+        p.presentation= @petition.presentation
         .row
           .col-xs-12
             = you_tube_embed @petition.video_id
         .row
           .col-xs-12
             h3.section-title.petition-body-title style="border-color: #{@cycle.color}" Projeto de lei
-        p.petition-body= markdown @petition.published_version.body
+        .petition-body= markdown @petition.published_version.body
```
Comment (guimello) [inline app/views/cycles/plugin_relations/petition.html.slim:127]: oh boy, it was inside a paragraph... nice

**Review by guimello — CHANGES_REQUESTED** [review]

File: app/assets/javascripts/mu.petition-signers.js (hunk 1 above)
Comment (viniciusvgv) [reply app/assets/javascripts/mu.petition-signers.js]: nice, thanks!

File: app/assets/stylesheets/petition/index.css.sass (hunk 4 above)
Comment (viniciusvgv) [reply app/assets/stylesheets/petition/index.css.sass:1]: :poop:

File: app/views/cycles/plugin_relations/petition.html.slim (hunk 5 above)
Comment (viniciusvgv) [reply app/views/cycles/plugin_relations/petition.html.slim:127]: yeah hahah and changing that was also the way to solve that crazy nesting error that I talked about on slack

File: app/assets/stylesheets/global.sass (hunk 3 above)
Comment (viniciusvgv) [reply app/assets/stylesheets/global.sass]: but useless lol the google fonts file already defines `.material-icons` so I could've just used it here. Fixed that on `1ed3e2a`

File: app/assets/stylesheets/petition/verify-documents.css.sass [hunk 6]
```diff
@@ -0,0 +1,246 @@
+@import "bootstrap/variables"
+
+#verify-document
+  padding: 60px 0 60px 0
```
Comment (guimello) [inline app/assets/stylesheets/petition/verify-documents.css.sass]: Check my PR https://github.com/itsriodejaneiro/mudamos-web/pull/238/files#diff-560d64c2e3a63e576d26439197448636L278

I removed some styles here

**Review by guimello — CHANGES_REQUESTED** [review]

**Review by guimello — APPROVED** [review]
🚀

**Pull request merged**